test/vscode-ui/test/helpers.ts

```diff
@@ -6,9 +6,9 @@ export async function switchToSubframe() {
   const iframe = await browser.$("iframe");
   await browser.switchToFrame(iframe);
 
-  await browser.$("iframe").waitForExist({ timeout: 3000 });
+  await browser.$("iframe").waitForExist({ timeout: 10000 });
```

Comment:
What was it that required a jump from 3 seconds to 10 seconds for the wait? (just interested, because that delay seems like something we would want to address.)

Reply:
I think I increased that up to 10 secs for troubleshooting. i can probably bump that back down.